src/Sylius/Bundle/WebBundle/Resources/translations/messages.en.yml

```diff
@@ -575,7 +575,7 @@ sylius:
         state: State
         immutable_warning: Already shipped orders are immutable.
         promotion_coupons: Coupons
-        no_promotion: No coupons were used.order:
+        no_promotion: No coupons were used.order
```

Comment:
This one looks like wrong merge, IMO should be `No coupons were used.`

@pjedrzejewski @michalmarcinkowski ping